src/Compilers/CSharp/Portable/BoundTree/BoundNodes.xml

```diff
@@ -1758,8 +1758,8 @@
   <Node Name="BoundRecursivePattern" Base="BoundPattern">
     <Field Name="DeclaredType" Type="BoundTypeExpression" Null="allow"/>
     <Field Name="DeconstructMethodOpt" Type="MethodSymbol" Null="allow"/>
-    <Field Name="Deconstruction" Type="ImmutableArray&lt;BoundPattern&gt;" Null="allow"/>
-    <Field Name="PropertiesOpt" Type="ImmutableArray&lt;(Symbol symbol, BoundPattern pattern)&gt;" Null="allow"/>
+    <Field Name="Deconstruction" Type="ImmutableArray&lt;BoundSubpattern&gt;" Null="allow"/>
```

Comment:
Not related to this PR: should `Deconstruction` be called `DeconstructionOpt`?

Reply:
We are inconsistent about that. I think we should do a sweep to some consistent state.